explore/src/main/scala/explore/EditableLabel.scala

```diff
@@ -139,13 +141,15 @@ object EditableLabel {
         if (editing.value.value)
           <.div(^.width := "100%", ^.display.flex)(
             InputText(
-              id = "editable-label-input", // won't necesessarily be unique...
+              id = s"editable-label-input-${Random.nextInt}",
               value = displayValue.value,
               onChange = (e: ReactEventFromInput) => displayValue.setState(e.target.value),
               clazz = props.inputClass
             ).mini.withMods(
               ^.onKeyUp ==> (e =>
-                if (e.key === "Enter") submitCB
+                // This is odd, it works fine in demo but not in explore
+                if (e.key === " ") displayValue.modState(_ + " ")
+                else if (e.key === "Enter") submitCB
```

Comment:
Really odd. Does it still happen if you comment out the whole `onKeyUp` parameter?

Reply:
Yes I tried all kind of things and this is the only thing it worked. I thought it could be related to prime-react but it works fine on the demo